### PR TITLE
AA: Add arbitrum testnet support

### PR DESCRIPTION
**Pull request by MatiNavCrossmint**
## Description

AA: Add arbitrum testnet support

## Test plan

Created AA Wallet on Arb sepolia
Mint NFT

### CODE REVIEW

File: packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts [hunk 1]
```diff
@@ -267,6 +269,8 @@ export function getViemNetwork(networkName: EVMBlockchainIncludingTestnet) {
             return astarZkEVM;
         case "optimism-sepolia":
             return optimismSepolia;
+        case "arbitrum-sepolia":
+            return arbitrumSepolia;
```
Comment (afeight) [inline packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts:273]: nit: can we move this to line 258 to group the arbitrum ones together?

**Review by afeight — APPROVED** [review]
looks good, small comment
there may really be a follow-up PR that sorts all of these lists tbh

File: packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts (hunk 1 above)
Comment (MatiNavCrossmint) [reply packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts]: I grouped all of them. LMK if looks good

**Review by afeight — APPROVED** [review]
thank you!

**Pull request merged**